.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,32 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
+        <value>
+          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
+        </value>
+      </option>
```

Comment:
```suggestion
```

The project doesn't use kotlin synthetic anymore

Reply:
Good catch! I made the change in the Android Studio GUI and committed the change in [`43a75bf` (#3672)](https://github.com/streetcomplete/StreetComplete/pull/3672/commits/43a75bfddd071751f0d000b9d7c6a1d68be2e18e).